about.html

```diff
@@ -12,57 +12,86 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+  <header>
+    <section class="topnav">
+        
+        <nav class="navmain">
+            <img src="/img/lambda-black.png" alt="">
+            <a href="index.html">Home</a>
+            <a href="about.html">About</a>
+            <a href="">Products</a>
+            <a href="">Blog</a>
+            <a href="">Contact</a>
+        </nav>
+        </section>
+        <section class="jumbohome">
+            <img src="img/jumbo-about.png" alt="" class="bigpic">
+        </section>
 
+</header>
+    
+     
+    <div class="container about-page">
+      <div class="abouttop">
 
-    <img src="img/about-plan.png" alt="strategy">
+     
+        <h1>About</h1>
     
```

Comment:
Your getting there on finishing the about.html.  Think about how you can wrap some of these areas in divs or semantic elements so you can target them with flexbox.